examples/PurePursuitController/Makefile

```diff
@@ -0,0 +1,20 @@
+NO_OPENCV := 1
+WITH_MATPLOTLIBCPP := 1
+WITH_SDL2 := 1
+include ../../make_common/bob_robotics.mk
```

Comment:
I've simplified the makefile system for BoB robotics now -- hang on and I'll push a simpler version of this to your branch